dpnp/backend/kernels/dpnp_krnl_mathematical.cpp

```diff
@@ -1227,104 +292,5 @@ void func_map_init_mathematical(func_map_t &fmap)
-    fmap[DPNPFuncName::DPNP_FN_REMAINDER][eft_FLT][eft_DBL] = {
-        eft_DBL, (void *)dpnp_remainder_default_c<double, float, double>};
-    fmap[DPNPFuncName::DPNP_FN_REMAINDER][eft_DBL][eft_INT] = {
-        eft_DBL, (void *)dpnp_remainder_default_c<double, double, int32_t>};
-    fmap[DPNPFuncName::DPNP_FN_REMAINDER][eft_DBL][eft_LNG] = {
-        eft_DBL, (void *)dpnp_remainder_default_c<double, double, int64_t>};
-    fmap[DPNPFuncName::DPNP_FN_REMAINDER][eft_DBL][eft_FLT] = {
-        eft_DBL, (void *)dpnp_remainder_default_c<double, double, float>};
-    fmap[DPNPFuncName::DPNP_FN_REMAINDER][eft_DBL][eft_DBL] = {
-        eft_DBL, (void *)dpnp_remainder_default_c<double, double, double>};
-
-    fmap[DPNPFuncName::DPNP_FN_TRAPZ][eft_INT][eft_INT] = {
-        eft_DBL, (void *)dpnp_trapz_default_c<int32_t, int32_t, double>};
-    fmap[DPNPFuncName::DPNP_FN_TRAPZ][eft_INT][eft_LNG] = {
-        eft_DBL, (void *)dpnp_trapz_default_c<int32_t, int64_t, double>};
-    fmap[DPNPFuncName::DPNP_FN_TRAPZ][eft_INT][eft_FLT] = {
-        eft_DBL, (void *)dpnp_trapz_default_c<int32_t, float, double>};
-    fmap[DPNPFuncName::DPNP_FN_TRAPZ][eft_INT][eft_DBL] = {
-        eft_DBL, (void *)dpnp_trapz_default_c<int32_t, double, double>};
-    fmap[DPNPFuncName::DPNP_FN_TRAPZ][eft_LNG][eft_INT] = {
-        eft_DBL, (void *)dpnp_trapz_default_c<int64_t, int32_t, double>};
-    fmap[DPNPFuncName::DPNP_FN_TRAPZ][eft_LNG][eft_LNG] = {
-        eft_DBL, (void *)dpnp_trapz_default_c<int64_t, int64_t, double>};
-    fmap[DPNPFuncName::DPNP_FN_TRAPZ][eft_LNG][eft_FLT] = {
-        eft_DBL, (void *)dpnp_trapz_default_c<int64_t, float, double>};
-    fmap[DPNPFuncName::DPNP_FN_TRAPZ][eft_LNG][eft_DBL] = {
-        eft_DBL, (void *)dpnp_trapz_default_c<int64_t, double, double>};
-    fmap[DPNPFuncName::DPNP_FN_TRAPZ][eft_FLT][eft_INT] = {
-        eft_DBL, (void *)dpnp_trapz_default_c<float, int32_t, double>};
-    fmap[DPNPFuncName::DPNP_FN_TRAPZ][eft_FLT][eft_LNG] = {
-        eft_DBL, (void *)dpnp_trapz_default_c<float, int64_t, double>};
-    fmap[DPNPFuncName::DPNP_FN_TRAPZ][eft_FLT][eft_FLT] = {
-        eft_FLT, (void *)dpnp_trapz_default_c<float, float, float>};
-    fmap[DPNPFuncName::DPNP_FN_TRAPZ][eft_FLT][eft_DBL] = {
-        eft_DBL, (void *)dpnp_trapz_default_c<float, double, double>};
-    fmap[DPNPFuncName::DPNP_FN_TRAPZ][eft_DBL][eft_INT] = {
-        eft_DBL, (void *)dpnp_trapz_default_c<double, int32_t, double>};
-    fmap[DPNPFuncName::DPNP_FN_TRAPZ][eft_DBL][eft_LNG] = {
-        eft_DBL, (void *)dpnp_trapz_default_c<double, int64_t, double>};
-    fmap[DPNPFuncName::DPNP_FN_TRAPZ][eft_DBL][eft_FLT] = {
-        eft_DBL, (void *)dpnp_trapz_default_c<double, float, double>};
-    fmap[DPNPFuncName::DPNP_FN_TRAPZ][eft_DBL][eft_DBL] = {
-        eft_DBL, (void *)dpnp_trapz_default_c<double, double, double>};
-
-    fmap[DPNPFuncName::DPNP_FN_TRAPZ_EXT][eft_INT][eft_INT] = {
```

Comment:
It is still used in `dpnp_trapz`. Are you sure that it would be safe to remove that?

Reply:
I cleaned up dpnp_trapz implementation, because it was never used.

Reply:
It's calling from `dpnp.trapz` but probably it's kind of dead code.
Then it would be great to clean up `dpnp_trapz`  everywhere: in c++ backend, cython and python implementations.

Reply:
Cleaned up dpnp_trapz implementation by last commit